libs/RobotKit/include/RobotController.h

```diff
@@ -196,7 +196,9 @@ class RobotController : public interface::RobotController
 	void startAutonomousActivityMode() final
 	{
 		_lcd.turnOn();
-		_behaviorkit.displayAutonomousActivitiesPrompt();
+
+		auto card = _rfidkit.getLastMagicCardActivated();
```

Comment:
this is smart but are you absolutely certain that it will work if:

- the user puts 2 cards (dice + something else)
- dice is read
- then the other one is read
- this function is called

?

Reply:
The execution speed between the dice_roll detection and the main menu display is so tiny that we couldn't put an other card immediately upon the dice roll detection, even willingly.